README.md

```diff
@@ -1,21 +1,52 @@
 # Identity Hub
```

Comment:
having a README.md at the root of the repository is necessary for GitHub to render it on the repo's landing page. IH is much less complex than the connector, so there are fewer links to documentation and I felt that having the key information on the landing page is useful.

EDC merely links from README.md -> docs/developer/README.md -> to other documents.

Reply:
ouch, I was wrong, in the sense, ok with the root README obviously, I thought it was a `docs/developer/README.md` document that's not existent in the `IdentityHub` but it could be good as entry point (or a simple index page) for dev docs. Something for a subsequent issue